lib/method_execution.rb

```diff
@@ -0,0 +1,27 @@
+require 'method_execution/array_destructer'
+
+module MethodExecution
```

Comment:
Not sure I get the naming here.  Why isn't this under the existing `module RuboClaus` namespace?   Could it not be shorted to just `RuboClaus::Method`.

Also having a module with only private methods in it is very odd and not idiomatic.